### ISSUE TITLE
Local haxelib server using sqlite uses different db path for website and remoting API

### ISSUE THREAD

**Issue by jasononeil** [HaxeFoundation/haxelib#56]
Using "localhost:2000/" for the website, and "localhost:2000/api/3.0/" for the API, they use different paths:

Website: /root/haxelib.db
API: /root/../../haxelib.db

This would obviously be a good idea to fix...

**Comment by back2dos** [comment]
Uhm, is this still an issue?

**Comment by jasononeil** [comment]
Yes, but I doubt it affects many people.  I'll look at it after some of the more urgent ones.

**Comment by nadako** [comment]
That is due to `-D haxelib_api`, but I really have no Idea why it's even needed. Let's remove it?:)

**Comment by back2dos** [comment]
Off the top of my head I think this is related to the fact that the
remoting API module is built separately from the web server and located in
some subfolder (api/3.0 or somesuch thing).

**Comment by nadako** [comment]
Maybe that's related to some mod_neko cwd behaviour? Because for nekotools server i have to compile WITHOUT `-D haxelib_api`

**Comment by nadako** [comment]
@andyli you do have access to haxelib servers nowadays? could you check if we can get rid of that `-D haxelib_api` define and not break anything? :)
